conda_build/convert.py

```diff
@@ -268,15 +268,19 @@ def update_lib_contents(lib_directory, temp_dir, target_platform, file_path):
         except IndexError:
             pass
 
+        # why not rename here?
         shutil.move(os.path.join(temp_dir, 'lib'), os.path.join(temp_dir, 'Lib'))
 
     elif target_platform == 'unix':
         temp_dir = Path(temp_dir)
         src_dir = temp_dir / "Lib"
+        intermediate_dir = temp_dir / "notLib"
```

Comment:
if this works we should use the technique when we are on a case-insensitive filesystem, which is usually true on Windows an Mac, and rarely true on Linux